pkg/kv/kvserver/allocator/storepool/store_pool.go

```diff
@@ -1063,9 +1108,9 @@ type StoreList struct {
 	// eligible to be rebalance targets.
 	candidateWritesPerSecond Stat
 
-	// candidateWritesPerSecond tracks L0 sub-level stats for Stores that are
-	// eligible to be rebalance targets.
-	CandidateL0Sublevels Stat
+	// CandidateIOOverload tracks the IO overload stats for Stores that are
+	// eligible to be rebalance candidates.
+	CandidateIOOverload Stat
```

Comment:
[nit] Rename this to CandidateIOOverloadScores to make the plural sense clearer.

Reply:
Updated.